src/Provider/Provider.Web/ViewModels/Reports/ReportDashboard/ReportsDashboardViewModel.cs

```diff
@@ -1,6 +1,31 @@
-﻿namespace Esfa.Recruit.Provider.Web.ViewModels.Reports.ReportDashboard
+﻿using System.Collections.Generic;
+using Esfa.Recruit.Vacancies.Client.Domain.Entities;
+using Humanizer;
+using Microsoft.EntityFrameworkCore.Internal;
```

Comment:
![Why](https://media.giphy.com/media/DvNwi41Iqrzos/giphy.gif)

Reply:
Arhhhhh!!!!!!!! `.Any` exists in `System.Linq` & `Microsoft.EntityFrameworkCore.Internal`. Will fix 👍